velox/functions/prestosql/Probability.h

```diff
@@ -43,5 +43,22 @@ struct BetaCDFFunction {
   }
 };
 
+static const double kSqrtOfTwo = sqrt(2);
```

Comment:
No global static in header, maybe you can put it inside `NormalCDFFunction`, or just calculate it in the function (`erf` will take most of the time anyway)

Reply:
@aditi-pandit earlier I had sqrt(2) as part of the  function. I moved it outside as global per your suggestion. Are you okay with me moving it back inside as part of function? 
cc @Yuhta

Reply:
Hi @poojabhandari : There seems some confusion here. I too had suggested to move the declaration in the function body to be closer to the logic. I totally agree with @Yuhta.

Reply:
Yeah just move the exact same line inside function body should work

Reply:
@Yuhta @aditi-pandit 
At very beginning I had sqrt(2) as part of the formula. To avoid confusion, could you please confirm if the following is fine ?


```
template <typename T>
struct NormalCDFFunction {
  VELOX_DEFINE_FUNCTION_TYPES(T);

  // Normal cumulative distribution is computed as per the reference at
  // https://mathworld.wolfram.com/NormalDistribution.html.

  const double kSqrtOfTwo = sqrt(2);

  FOLLY_ALWAYS_INLINE void
  call(double& result, double m, double sd, double value) {
    VELOX_USER_CHECK_GT(sd, 0, "standardDeviation must be > 0");

    result = 0.5 * (1 + erf((value - m) / (sd * kSqrtOfTwo)));
  }
};
```